dev/how-to/migrate/full-from-mysql.md

```diff
@@ -0,0 +1,124 @@
+---
+title: 全量数据迁移
+category: how-to
+---
```

Comment:
```suggestion
aliases: ['/docs-cn/dev/how-to/migrate/from-mysql/']
---
```
When you relocate a file, you have to add an alias for it to redirect the old link.